packages/pnpm/src/main.ts

```diff
@@ -124,17 +124,24 @@ export default async function run (inputArgv: string[]) {
       process.exit(0)
       return
     }
-    config.selectedProjectsGraph = await filterPackages(allProjects, config.filter ?? [], {
+    const filterResults = await filterPackages(allProjects, config.filter ?? [], {
       prefix: process.cwd(),
       workspaceDir: wsDir,
     })
+    config.selectedProjectsGraph = filterResults.selectedProjectsGraph
     if (R.isEmpty(config.selectedProjectsGraph)) {
       if (!config['parseable']) {
         console.log(`No projects matched the filters in "${wsDir}"`)
       }
       process.exit(0)
       return
```

Comment:
Do we need a return after the `process.exit(0)`? is it reachable code?

Reply:
not needed but the linter is silly, so it will report a warning if you remove it